### DISCUSSION TITLE
Don't make EE7 upgrade look like a critical update.

### DISCUSSION THREAD

**Discussion by appricosa** [ExpressionEngine/ExpressionEngine#2904] [Feature Requests]
I am running 6.5.3 and when someone clicks on the update/version badge in the bottom left corner, they get alerted by a very aggressive upgrade notice.

<img width="433" alt="Bildschirmfoto 2022-08-28 um 15 17 55" src="https://user-images.githubusercontent.com/49977166/187075930-d5c9adf0-d028-4d09-b2e3-a0e59850a245.png">

Since EE6 is LTS, it does not make sense to have the upgrade to EE7 message look like it is super critical and essential security update.

How will updates for EE6 appear in the future, when someone would like to stick with EE 6 for now?
Will the EE7 upgrade remain the entire time, or will there be an option to just receive EE6 LTS updates?

**Comment by intoeetive** [comment]
@appricosa thank you for the feedback

Upgrading to v7 is only possible if you are running the latest (for the current moment) version of EE6

That means, as soon as new version of EE6 will be released, you will it as available update (and not v7)
